Task 1/Assets/Scripts/EventController.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+
+public class EventController
```

Comment:
and btw - you can define this class as static class(please read more about static classes online)